src/core/server/ui_settings/ui_settings_client.ts

```diff
@@ -72,17 +69,25 @@ export class UiSettingsClient implements IUiSettingsClient {
 
   async get<T = any>(key: string): Promise<T> {
     const all = await this.getAll();
-    return all[key];
+    return all[key] as T;
   }
 
   async getAll<T = any>() {
-    const raw = await this.getRaw();
+    const result: Record<string, any> = {};
+
+    Object.keys(this.defaults).forEach((key) => {
+      result[key] = this.defaults[key].value;
+    });
```

Comment:
Q: does it make sense to have this value pre-set in the constructor to avoid calling it every time?

Reply:
done in https://github.com/elastic/kibana/pull/85027/commits/40b33648b0ad9c2e82c3b87f1c161ba1c4d71b45